__tests__/app.test.js

```diff
@@ -144,10 +144,59 @@ describe("app", () => {
     });
   });
  });
+ });
+ describe("GET /api/articles/:article_id/comments", () => {
+  test("200: responds with a status of 200 when the comments for an article is found", () => {
+    const testArticleId = 1;
```

Comment:
You shouldn't have a test just for a status code. Lower down you also have a happy path test asserting a 200, this is surplus to requirement.